packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md

```diff
@@ -82,7 +82,7 @@ The OAuth 2.0 Client Secret method sends a signed JWT to your external service.
 * Create an app integration
 * Add a custom scope
 * Add OAuth 2.0 authentication fields to your inline hook
-* Add code to verify the JWT
+* Add code to your external service to verify the JWT
```

Comment:
Aren't these list items complete sentences in imperative form? They should have a period at the end of them.